### PR TITLE
refactor: refine code and wording for PR #796

### PR DESCRIPTION
**Pull request by cheton**

### CODE REVIEW

File: src/app/store/index.js [hunk 1]
```diff
@@ -183,6 +183,15 @@ const migrateStore = () => {
     if (semver.lt(cnc.version, '1.9.16')) {
         store.unset('widgets.axes.jog.step');
     }
+
+    // 1.9.28
```
Comment (emcniece) [inline src/app/store/index.js:187]: I might have messed up the last release 😬 apologies.

https://github.com/cncjs/cncjs/releases/tag/v1.9.28 already exists, but I failed to increment package.json accordingly. Some documentation for release procedure would help, but that's on me.

If you wish to target the next release, 1.9.29 might be better.

File: src/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cncjs-app",
-  "version": "1.9.26",
+  "version": "1.9.27",
```
Comment (emcniece) [inline src/package.json:3]: Same as noted above - apologies, I failed the version update procedure.

**Review by emcniece — APPROVED** [review]

File: src/app/store/index.js (hunk 1 above)
Comment (cheton) [reply src/app/store/index.js:187]: The official version of CNCjs is determined by publishing a release on NPM. You can view the available versions on the following link: 

https://www.npmjs.com/package/cncjs?activeTab=versions
image

As the recent changes do not include any new features and there is no commit for version bump, I propose using current version 1.9.28 for the next release to include PR #796 and PR #800.

File: src/package.json (hunk 2 above)
Comment (cheton) [reply src/package.json:3]: It's ok. I will proceed to update the version field as per usual. 😃

**Pull request merged**